src/main/resources/assets/waystones/lang/de_DE.lang

```diff
@@ -0,0 +1,32 @@
+tile.waystones:waystone.name=Wegstein
+
+item.waystones:warp_scroll.name=Warprolle
+item.waystones:return_scroll.name=Rückkehrrolle
+item.waystones:warp_stone.name=Warpstein
+
+itemGroup.waystones=Wegsteine
+
+waystones:noDimensionWarp=Du kannst nicht zwischen Welten springen.
+waystones:waystoneBroken=Der verbundene Wegstein wurde bewegt oder zerstört.
+waystones:scrollNotBound=Du hast bisher noch keine Wegsteine aktiviert.
+waystones:stoneNotCharged=Der Warpstein ist noch nicht vollständig aufgeladen.
+waystones:activatedWaystone=Wegstein aktiviert: %s
+waystones:nameOccupied=Der Name %s wird bereits verwendet.
+waystones:notTheOwner=Wegsteine können nur von ihrem Errichter umbenannt werden.
+waystones:creativeRequired=Du musst im Kreativmodus sein um einen globalen Wegstein zu bearbeiten.
+
+gui.waystones:warpStone.selectDestination=Wähle dein Ziel:
+gui.waystones:warpStone.nextPage=Nächste
+gui.waystones:warpStone.previousPage=Vorherige
+gui.waystones:confirmReturn=Möchtest du wirklich zu deinem letzen Wegstein zurückspringen?
+gui.waystones:confirmReturn.boundTo=Verbunden mit: %s
+gui.waystones:confirmReturn.noWaystoneActive=Kein Wegstein aktiv.
+gui.waystones:editWaystone.enterName=Gebe den Namen ein:
+gui.waystones:editWaystone.isGlobal=Ist Global
+
+tooltip.waystones:cooldownLeft=Abkühlung: %d Sekunden
+tooltip.waystones:boundTo=Verbunden mit: %s
+tooltip.waystones:none=Keiner
```

Comment:
This one is used with `tooltip.waystones:boundTo` so it should probably be "keinen"? (lower case intended)